### DISCUSSION TITLE
Annoying system beep when booting

### DISCUSSION THREAD

**Discussion by quadransmuralis** [SpiralLinux/SpiralLinux-project#100] [Q&A]
Hello,
When booting Spiral Linux with the Gnome desktop from a live usb there is an annoying beep.
Is there a way to stop this deafening behaviour?
Thx

**Comment by geckolinux** [comment]
Hi there, sorry for the slow reply. Could you please confirm if the beep happens on a system with legacy BIOS or UEFI?

**Comment by quadransmuralis** [comment]
UEFI, but only with a live usb. Once installed, there is no beep!
Thank you!

On Tue, 30 Aug 2022 at 16:14, GeckoLinux ***@***.***> wrote:

> Hi there, sorry for the slow reply. Could you please confirm if the beep
> happens on a system with legacy BIOS or UEFI?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SpiralLinux/discussions/100#discussioncomment-3509450>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFP6GXW2VKK2UNYRF2ZWKTTV3YJL5ANCNFSM57D4CI7Q>
> .
> You are receiving this because you authored the thread.Message ID:
> <SpiralLinux/SpiralLinux-project/repo-discussions/100/comments/3509450@
> github.com>
>


-- 
Robert Heath
+33 6 50 75 73 95

**Reply by geckolinux** [reply]
OK, thanks for confirming, I'll see if there's a clean way to disable that for the next release.

**Comment by geckolinux** [comment]
Removed in the SpiralLinux 11.220925 release.
